src/test/java/pages/HomePage.java

```diff
@@ -4,63 +4,51 @@
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
-import org.testng.Assert;
-
-import java.time.Duration;
 
 public class HomePage extends BasePage{
 
-    //Locators
-    By avatarIcon = By.cssSelector("img.avatar");
-    By successMsg = By.cssSelector("div.success.show");
-    By plusButton = By.cssSelector("[class='fa fa-plus-circle create']");
-    By newPlaylistOption = By.cssSelector("li[data-testid='playlist-context-menu-create-simple']");
-    By playlistInputField = By.cssSelector("input[name='name']");
-    By allSongsButton = By.xpath("//a[contains(text(), 'All Songs')]");
-    By playlist = By.cssSelector(".playlist:nth-child(3)");
+    //WebElements
+    @FindBy (css="[class='fa fa-plus-circle create']")
+    WebElement createPlaylistButton;
+    @FindBy (css="li[data-testid='playlist-context-menu-create-simple']")
+    WebElement newPlaylistOption;
+    @FindBy (css="input[name='name']")
+    WebElement playlistInputField;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

  
  
```
    @FindBy (css="[type='email']")
    private WebElement emailField;
    @FindBy (css="[type='password']")
    private WebElement passwordField;
    @FindBy (css="button[type='submit']")
    private WebElement submitButton;
```